### ISSUE TITLE
Wrong APK in app/release?

### ISSUE THREAD

**Issue by IzzySoft** [imamhossain94/GestureVolume#1]
I just wanted to integrate your app with [my repo](https://apt.izzysoft.de/fdroid) and thus pulled the APK from above location. But other than the name of the repo and the code inside it plus the link to PlayStore suggest, it seems to be something else:

    package: name='com.newagedevs.couplewidgets' versionCode='1' versionName='1.0.0'

which probably comes [from here](https://github.com/imamhossain94/CoupleWidgetsMVVM). Is there a chance you could [tag your releases](https://github.com/imamhossain94/GestureVolume/releases) and attach the correct APK there? Thanks in advance!

**Comment by imamhossain94** [comment]
Yes, you’re right. The APK in app/release was incorrect. The correct APK is:

package: name='com.newagedevs.gesturevolume' versionCode='31' versionName='1.3.1'

Sorry for the incorrect APK and for the very late response. Thanks for pointing this out!
